fitnesse/resources/bootstrap-plus/js/bootstrap-plus-extra.js

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Is er een specifieke reden deze functies in een aparte js op te nemen i.p.v. in de algeene bootstrap-plus?

Reply:
overzichtelijkheid, het idee was om stukken code doe wij schreven in een tijdelijke file te zetten en dit later op een slimmere manier aan te pakken aan de hand van de hoeveelheid regels die we hebben geschreven voor bepaalde functies